src/app/components/favorites/favorites.component.ts

```diff
@@ -0,0 +1,30 @@
+// favorites.components.ts
+
+import { Component, OnInit } from '@angular/core';
+import { JokeService } from '../../joke.service';
+
+@Component ({
+  selector: 'app-favorites',
+  templateUrl: './favorites.component.html',
+  styleUrls: ['./favorites.component.css']
+})
+export class FavoritesComponent implements OnInit {
+
+  constructor(private jokeService: JokeService) { }
+
+  // get favorites array from service
+get favData(): any {
```

Comment:
Class functions should be indented one tab to show better visual hierarchy with the class as a developer reads through the code. This practice also helps IDEs and editors with code folding.